.github/workflows/build.yml

```
@@ -464,7 +464,11 @@ jobs:
         with:
           components: clippy
       # We want to be free of any warnings, so deny them.
-      - run: ./admin/clippy -- --deny warnings
+      # - Allow `clippy::disallowed_types` as a workaround since `disallowed_types` configured in `.clippy.toml`
+      #   is only intended for main `rustls` crate.
```

minor nit, I will fix this while rebasing:

```suggestion
      #   is only intended for the main `rustls` crate.
```